Client/Frontend/Browser/HomePanel/NTPDownloader.swift

```diff
@@ -328,7 +328,7 @@ class NTPDownloader {
             }
             
             if response.statusCode != 304 && (response.statusCode < 200 || response.statusCode > 299) {
-                completion(nil, nil, "Invalid Response Status Code: \(response.statusCode)")
+                return completion(nil, nil, "Invalid Response Status Code: \(response.statusCode)")
```

Comment:
This return should be on a separate line, since we aren't actually returning anything. 
```suggestion
                completion(nil, nil, "Invalid Response Status Code: \(response.statusCode)")
                return
```